From 168ae35bde46a713bf08cbb4bc3456a427531e32 Mon Sep 17 00:00:00 2001
From: Christopher Degawa <ccom@randomderp.com>
Date: Wed, 13 May 2020 19:53:28 +0000
Subject: [PATCH] include/medialibrary/IDeviceLister.h: Include string

`std::string` is used, but string is not included, causing errors when compiling with mingw-w64
---
 include/medialibrary/IDeviceLister.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/medialibrary/IDeviceLister.h b/include/medialibrary/IDeviceLister.h
index f6324610..299e5995 100644
--- a/include/medialibrary/IDeviceLister.h
+++ b/include/medialibrary/IDeviceLister.h
@@ -24,6 +24,7 @@
 
 #include <tuple>
 #include <vector>
+#include <string>
 
 namespace medialibrary
 {
-- 
2.26.0.windows.1

